batch2/batch/driver/instance_pool.py

```diff
@@ -18,7 +18,7 @@ class InstancePool:
     def __init__(self, driver):
         self.driver = driver
         self.worker_type = WORKER_TYPE
-        self.worker_cores = WORKER_CORES
+        self.worker_cores = WORKER_CORES * 1000  # mCPU
```

Comment:
Rather than a comment, when a variable has a unit that isn't totally obvious from context, I prefer to suffix the variable name with the unit: `worker_cores_mcpu` (like `worker_disk_size_gb` elsewhere in this file).  This obviates the comment and makes the units clear in other places, not just when you're multiplying by 1000.